src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -629,17 +644,29 @@ else if (outFor.equals("GATK")) {
 						} else if (outFor.equals("TXT")) usage("Output format 'TXT' has been deprecated. Please use 'VCF' instead.\nYou can extract VCF fields to a TXT file using 'SnpSift extractFields' (http://snpeff.sourceforge.net/SnpSift.html#Extract).");
 						else usage("Unknown output file format '" + outFor + "'");
 					}
-				} else if ((arg.equals("-s") || arg.equalsIgnoreCase("-stats"))) {
+				} else if (arg.equalsIgnoreCase("-htmlStats")) {
+					createSummaryHTML = true;
 					if ((i + 1) < args.length) {
-						summaryFile = args[++i];
-						String base = Gpr.baseName(Gpr.baseName(summaryFile, ".html"), ".csv"); // Extension can be either HTML or CSV
-						String dir = Gpr.dirName(summaryFile);
-						summaryGenesFile = (dir != null ? dir + "/" : "") + base + ".genes.txt";
+						summaryFileHTML = args[++i];
+                        if (summaryGenesFile.equals(DEFAULT_SUMMARY_GENES_FILE)) {
+						    String base = Gpr.baseName(summaryFileHTML, ".html"); 
+						    String dir = Gpr.dirName(summaryFileHTML);
+						    summaryGenesFile = (dir != null ? dir + "/" : "") + base + ".genes.txt";
+                        }
 					}
-				} else if (arg.equalsIgnoreCase("-noStats")) createSummary = false; // Do not create summary file. It can be much faster (e.g. when parsing VCF files with many samples)
+				} else if ((arg.equals("-s") || arg.equalsIgnoreCase("-stats"))) { 
+					usage("Input format -s and -stats have been deprecated. Please use -htmlStats or -csvStats instead.");
```

Comment:
Probably better to leave the default behaviour, which enables HTML summary.
This 'if' statement can be collapsed with the one parsing htmlStats